github-pages-health-check.gemspec

```diff
@@ -1,28 +1,30 @@
-require File.expand_path("../lib/github-pages-health-check/version", __FILE__)
+# frozen_string_literal: true
+require File.expand_path('../lib/github-pages-health-check/version', __FILE__)
 
 Gem::Specification.new do |s|
-  s.required_ruby_version = ">= 1.9.3"
+  s.required_ruby_version = '>= 1.9.3'
 
-  s.name                  = "github-pages-health-check"
+  s.name                  = 'github-pages-health-check'
   s.version               = GitHubPages::HealthCheck::VERSION
-  s.summary               = "Checks your GitHub Pages site for commons DNS configuration issues"
-  s.description           = "Checks your GitHub Pages site for commons DNS configuration issues."
-  s.authors               = "GitHub, Inc."
-  s.email                 = "support@github.com"
-  s.homepage              = "https://github.com/github/github-pages-health-check"
-  s.license               = "MIT"
+  s.summary               = 'Checks your GitHub Pages site for commons DNS configuration issues'
```

Comment:
I think it's our rule to always use double quotes, not single quotes. I'd borrow from the `github/pages-gem` rubocop.yml and make adjustments as necessary.